tests/index.ts

```diff
@@ -337,497 +350,502 @@ describe('isSameMinute, isSameHour, isSameDay, isSameMonth, isSameYear', () => {
   });
 
   it('returns interval string for 5 hours', () => {
     const date = new Date();
     date.setHours(date.getHours() - 5);
 
-    expect(getRelativeDate(date)).to.equal('5 часов');
+    expect(formatDistanceToNowStrict(date)).to.equal('5 часов');
   });
 
   it('returns interval string for 1 day', () => {
     const date = new Date();
     date.setDate(date.getDate() - 1);
 
-    expect(getRelativeDate(date)).to.equal('день');
+    expect(formatDistanceToNow(date)).to.equal('1 день');
   });
 
   it('returns interval string for 2 days', () => {
     const date = new Date();
     date.setDate(date.getDate() - 2);
 
-    expect(getRelativeDate(date)).to.equal('2 дня');
+    expect(formatDistanceToNow(date)).to.equal('2 дня');
   });
 
   it('returns interval string for 5 days', () => {
     const date = new Date();
     date.setDate(date.getDate() - 5);
 
-    expect(getRelativeDate(date)).to.equal('5 дней');
+    expect(formatDistanceToNow(date)).to.equal('5 дней');
   });
 
   it('returns interval string for 1 month', () => {
     const date = new Date();
     date.setMonth(date.getMonth() - 1);
 
-    expect(getRelativeDate(date)).to.equal('месяц');
+    expect(formatDistanceToNow(date)).to.equal('около 1 месяца');
+    expect(formatDistanceToNowStrict(date)).to.equal('1 месяц');
   });
 
   it('returns interval string for 2 months', () => {
     const date = new Date();
     date.setMonth(date.getMonth() - 2);
 
-    expect(getRelativeDate(date)).to.equal('2 месяца');
+    expect(formatDistanceToNowStrict(date)).to.equal('2 месяца');
   });
 
   it('returns interval string for 5 months', () => {
     const date = new Date();
     date.setMonth(date.getMonth() - 5);
 
-    expect(getRelativeDate(date)).to.equal('5 месяцев');
+    expect(formatDistanceToNowStrict(date)).to.equal('5 месяцев');
   });
 
   it('returns interval string for 1 year', () => {
     const date = new Date();
     date.setFullYear(date.getFullYear() - 1);
 
-    expect(getRelativeDate(date)).to.equal('год');
+    expect(formatDistanceToNow(date)).to.equal('около 1 года');
+    expect(formatDistanceToNowStrict(date)).to.equal('1 год');
   });
 
   it('returns interval string for 2 years', () => {
     const date = new Date();
     date.setFullYear(date.getFullYear() - 2);
 
-    expect(getRelativeDate(date)).to.equal('2 года');
+    expect(formatDistanceToNowStrict(date)).to.equal('2 года');
   });
 
   it('returns interval string for 5 years', () => {
     const date = new Date();
     date.setFullYear(date.getFullYear() - 5);
 
-    expect(getRelativeDate(date)).to.equal('5 лет');
+    expect(formatDistanceToNowStrict(date)).to.equal('5 лет');
   });
 });
 
 describe('getUtcOffset', () => {
   it('returns UTC offset in hours', () => {
-    expect(getUtcOffset(new Date(2020, 0, 1))).to.equal(+localUTCOffset);
+    const timezone = format(new Date(), 'xxx');
+    expect(millisecondsToHours(getTimezoneOffset(timezone))).to.equal(+localUTCOffset);
   });
```

Comment:
There's no possibility to get timezone offset as a number with `date-fns`. `getTimezoneOffset` is a helper from [date-fns-tz](https://github.com/marnusw/date-fns-tz#gettimezoneoffset).